packages/core/src/utilities/VoxelManager.ts

```diff
@@ -790,8 +801,11 @@ export default class VoxelManager<T> {
     };
 
     voxelManager._getScalarDataLength = () => {
-      const { pixelData } = getPixelInfo(0);
-      return pixelData.length * dimensions[2];
+      const { voxelManager: imageVoxelManager } = getPixelInfo(0);
+      if (!imageVoxelManager) {
```

Comment:
Is there a case where there wouldn't be a `voxelManager`? Is it safe to return `0` if it doesn't exist.